pkg/gadgets/top/block-io/tracer/tracer.go

```diff
@@ -54,6 +54,7 @@ type Tracer struct {
 	eventCallback    func(*top.Event[types.Stats])
```

Comment:
Mmmm I was wondering what we should do if the interval is greater than the timeout. Should it return an error? With this PR, it will print what it has collected when the timeout is reached. WDYT?

Reply:
Agree. Applied!